src/client/src/apps/MainRoute/widgets/spotTile/components/notifications/NotificationContainer.tsx

```diff
@@ -45,7 +47,7 @@ export default class NotificationContainer extends PureComponent<Props> {
       const { dealtCurrency, tradeId } = lastTradeExecutionStatus.trade
       const { terms } = currencyPair
       if (lastTradeExecutionStatus.trade.status === TradeStatus.Done) {
-        const { direction, notional, spotRate, tradeDate } = lastTradeExecutionStatus.trade
+        const { direction, notional, spotRate } = lastTradeExecutionStatus.trade
```

Comment:
Can't we just pull valueDate out of the last trade execution status here, like was done with tradeDate before? I think (though I may be wrong), that valueDate and tradeDate should both be present.

Reply:
cool, let me check